src/content/docs/reference/recipe.mdx

```diff
@@ -49,6 +61,23 @@ The tag of the base image to build on. Used to select a version explicitly (`40`
 image-version: 40
 ```
 
+### `stages:`
+
+A list of [stages](/references/stages/) that are executed before the build of the final image. This is useful for compiling programs from source without polluting the final bootable image.
+
+#### Example:
+
+```yaml
+stages:
+  - name: bluebuild
+    from: rust:1.77
```

Comment:
Or should this be `docker.io/library/rust:1.77`? Is it a risk that some podman environment wouldn't know where to look for this image?

Reply:
Oh you're right. I forgot podman does that